scripts/checkpatch_inc.sh

```diff
@@ -7,9 +7,14 @@ CHECKPATCH_IGNORE=$(echo core/lib/lib{fdt,tomcrypt} lib/lib{png,utils,zlib} \
 _CP_EXCL=$(for p in $CHECKPATCH_IGNORE; do echo ":(exclude)$p" ; done)
 
 function _checkpatch() {
+		# Use typedefsfile if supported by the checkpatch scripts
+		a="--typedefsfile typedefs.checkpatch"
```

Comment:
Ok, I'll go for this `... --help | grep ...` test. I did nit like it initially, but thinking back, it's a bit more reliable that my proposal.